spec/unit/backtracie_spec.rb

```diff
@@ -475,7 +593,9 @@ def it.test_method
       it_should_behave_like "an equivalent of the Ruby API (using locations)"
 
       it do
-        expect(backtracie_stack[2].qualified_method_name).to eq "Class$singleton.test_method"
+        # This used to say Class$singleton, but Object$singleton is more correct I believe - this is the singleton class of an object
+        # of type object.
```

Comment:
This comment doesn't seem accurate anymore?

Reply:
Oh yeah I renamed Object to ClassToGetASingletonMethodDefined after writing that comment because the comment confused me too so I figured I'd make the code unambiguous.